synfig-core/src/modules/mod_geometry/advanced_outline.cpp

```diff
@@ -872,18 +858,3 @@ Advanced_Outline::connect_bline_to_wplist(etl::loose_handle<ValueNode> x)
 	wplist->set_bline(ValueNode::Handle(x));
 	return true;
 }
-
-bool
-Advanced_Outline::connect_bline_to_dilist(etl::loose_handle<ValueNode> x)
```

Comment:
I didn't check the logic, but it looks like dilist is being used in Advanced outlines.

Can they be safely removed?

**tool_skeleton_icon.sif**
```
<layer type="advanced_outline" active="true" exclude_from_rendering="false" version="0.3" desc="outline">
...

<param name="dilist">
  <dilist type="dash_item" loop="false">
    <entry>
      <composite guid="85AA7300D0817205EA3332A673956D17" type="dash_item">
        <offset>
          <real value="0.1000000000"/>
        </offset>
        <length>
          <real value="0.1000000000"/>
        </length>
        <side_before>
          <integer value="4"/>
        </side_before>
        <side_after>
          <integer value="4"/>
        </side_after>
      </composite>
    </entry>
  </dilist>
</param>
<param name="dash_offset">
  <real value="0.0000000000"/>
</param>
```

Reply:
The code set bline, but this is never read

Reply:
Ok. Please rebase then.

Reply:
Rebased.